Added 'I love git' ascii art for git lov typo

avanzzzi opened the pull request:
Closes #19

avanzzzi committed:
Added 'I love git' ascii art for git lov typo

Xaxetrov commented:
Thanks for your contribution ! I can have a look at it on sunday

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -137,6 +137,13 @@ gitTypoStatue="
+╚╗╔╝
+╔╝(¯`v´¯)
+╚══`.¸.[GIT]
+'
```
I don't really get the heart. Where is it ?

avanzzzi replied:
Changed to something that looks like a heart now.

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -282,6 +289,12 @@ git() {
+
+  # log
+  elif [ "$1" = "lov" ]; then
+    clear; echo "$gitTypoHeart"
```
Please do not clear the shell if the ascii art is not dynamic. Instead just display the image like it is done for "git statue"

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -282,6 +289,12 @@ git() {
     echo "$gitTypoStatue"
     command git status "${@:2}"
+
+  # log
```
Please keep the easter eggs ordered alphabetically.

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -137,6 +137,13 @@ gitTypoStatue="
 /                     (_
 "
 
+gitTypoHeart='
```
please keep ascii art constants ordered alphabetically.

avanzzzi committed:
Changed heart ascii art
Sorted easter eggs and ascii art constants alphabetically
Removed clear shell in git lov typo

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -87,6 +87,23 @@ gitTypoExplosion7='
      ) /
     ,---.'
     
+gitTypoHeart='
```
Haha nice so big.

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -238,6 +255,12 @@ git() {
+  # log
+  elif [ "$1" = "lov" ]; then
+    echo "$gitTypoHeart"
+    sleep 1.5
```
Not sure about the sleep but we will keep it and I'll see later if I remove it or add it on other ascii arts.
